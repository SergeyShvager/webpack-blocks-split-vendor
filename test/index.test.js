import { webpack } from '@webpack-blocks/webpack2'
import splitVendor from '../src'

describe('splitVendor', () => {
  let context

  beforeEach(() => {
    context = { webpack }
  })

  it('returns webpack block with passed chunkName', () => {
    expect(splitVendor('foo')(context)).toMatchSnapshot()
  })

  it('returns webpack block with default chunkName', () => {
    expect(splitVendor()(context)).toMatchSnapshot()
  })
})
