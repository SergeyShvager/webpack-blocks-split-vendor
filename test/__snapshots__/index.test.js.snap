// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`splitVendor returns webpack block with default chunkName 1`] = `
Object {
  "output": Object {
    "filename": "[name].[chunkhash].js",
  },
  "plugins": Array [
    CommonsChunkPlugin {
      "async": undefined,
      "children": undefined,
      "chunkNames": Array [
        "vendor",
      ],
      "filenameTemplate": undefined,
      "ident": "/Users/diegohaz/Projects/webpack-blocks-split-vendor/node_modules/webpack/lib/optimize/CommonsChunkPlugin.js1",
      "minChunks": [Function],
      "minSize": undefined,
      "selectedChunks": undefined,
    },
    WebpackMd5Hash {},
  ],
}
`;

exports[`splitVendor returns webpack block with passed chunkName 1`] = `
Object {
  "output": Object {
    "filename": "[name].[chunkhash].js",
  },
  "plugins": Array [
    CommonsChunkPlugin {
      "async": undefined,
      "children": undefined,
      "chunkNames": Array [
        "foo",
      ],
      "filenameTemplate": undefined,
      "ident": "/Users/diegohaz/Projects/webpack-blocks-split-vendor/node_modules/webpack/lib/optimize/CommonsChunkPlugin.js0",
      "minChunks": [Function],
      "minSize": undefined,
      "selectedChunks": undefined,
    },
    WebpackMd5Hash {},
  ],
}
`;
